packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -117,7 +139,13 @@ export default function CreatePatternModal( {
 							{ __( 'Cancel' ) }
 						</Button>
 
-						<Button variant="primary" type="submit">
+						<Button
+							variant="primary"
+							type="submit"
+							disabled={ isSaving }
+							aria-disabled={ isSaving }
```

Comment:
a11y suggestion: we should only use `aria-disabled` here to allow focus on disabled elements. This means that we have to return early in `onCreate` when `isSaving` is true since `aria-disabled` doesn't prevent click actions.

You can see a similar pattern in action here: https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-post/src/components/sidebar/post-template/create-modal.js

Reply:
I also noticed this and fixed it in [f0ee899](https://github.com/WordPress/gutenberg/pull/54676/commits/f0ee8990cb6843ad91042c6d4824496e7539c27c) 👍